core/src/main/scala/com/ing/baker/compiler/RecipeValidations.scala

```diff
@@ -136,9 +136,9 @@ object RecipeValidations {
       .map(c => s"Event class: $c is not serializable by akka")
 
     val ingredientSerializationErrors: Seq[String] =
-      compiledRecipe.ingredients.values
-        .filterNot(hasAkkaSerializer)
-        .map(c => s"Ingredient class: $c is not serializable by akka").toSeq
+      compiledRecipe.ingredients
+        .filterNot{case (c, v) => hasAkkaSerializer(v) }
```

Comment:
- `c` is not used anywhere. consider using `_`
-  could you please add spaces before and after the `{`